tests/common/dualtor/dual_tor_utils.py

```diff
@@ -87,4 +90,55 @@ def get_t1_ptf_ports(dut, tbinfo):
         ptf_portchannel_intfs.append(intf_name)
 
     logger.info("Using portchannel ports {} on PTF for DUT {}".format(ptf_portchannel_intfs, dut.hostname))
-    return ptf_portchannel_intfs 
+    return ptf_portchannel_intfs
+
+
+def update_mux_configs_and_config_reload(dut, state):
+    """
+    @summary: Update config_db.json, and then load with 'config reload'
+            Please note that this is a general method, and caller must
+            backup config_db.json and do a restore at the end.
+    @param dut: The DUT we are testing against
+    @param state: A str, auto|active|standby
+    """
+    STATE_LIST = ['auto', 'active', 'standby']
+    pytest_assert(state in STATE_LIST, "state should be one of {}".format(STATE_LIST))
+
+    mux_cable_config = dut.shell("sonic-cfggen -d  --var-json 'MUX_CABLE'")['stdout']
+    pytest_assert(len(mux_cable_config.strip()) != 0, "No mux_cable configuration is found in config_db")
+
+    # Update mux_cable state and dump to a temp file
+    mux_cable_config_json = json.loads(mux_cable_config)
+    for _, config in mux_cable_config_json.items():
+            config['state'] = state
+    mux_cable_config_json = {"MUX_CABLE": mux_cable_config_json}
+    TMP_FILE = "/tmp/mux_config.json"
+    with open(TMP_FILE, "w") as f:
+        json.dump(mux_cable_config_json, f)
+    
+    dut.copy(src=TMP_FILE, dest=TMP_FILE)
+    
+    # Load updated mux_cable config with sonic-cfggen
+    cmds = [
+        "sonic-cfggen -j {} -w".format(TMP_FILE),
+        "config save -y"
+    ]
+    dut.shell_cmds(cmds=cmds)
+    config_reload(dut)
+    dut.file(path=TMP_FILE, state='absent')
+
+
+def force_active_tor(dut, intf):
```

Comment:
I would suggest to add a generic method toggle_tor_intf_state to force a particular intf to active or standby

Reply:
It's a good suggestion. But we are not able to toggle a DUT to standby state currently. We can only set the muxcable state to ```auto``` or ```active```.